File: app/src/main/java/com/imperium/academio/ui/adapters/helper/ClassViewHolder.java
package com.imperium.academio.ui.adapters.helper;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.constraintlayout.widget.ConstraintLayout;
import androidx.recyclerview.widget.RecyclerView;

import com.imperium.academio.R;

public class ClassViewHolder extends RecyclerView.ViewHolder {
    public final TextView name;
    public final ConstraintLayout constraintLayout;

    public ClassViewHolder(@NonNull View itemView) {
        super(itemView);
        name = itemView.findViewById(R.id.class_item_name);
        constraintLayout = itemView.findViewById(R.id.class_register_rvl);
    }
}

File: app/src/main/java/com/imperium/academio/ui/adapters/helper/ClassViewHolder.kt
package com.imperium.academio.ui.adapters.helper

import android.view.View
import android.widget.TextView
import androidx.constraintlayout.widget.ConstraintLayout
import androidx.recyclerview.widget.RecyclerView
import com.imperium.academio.R

class ClassViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @JvmField
    val name: TextView = itemView.findViewById(R.id.class_item_name)

    @JvmField
    val constraintLayout: ConstraintLayout = itemView.findViewById(R.id.class_register_rvl)

}